dev/tests/integration/testsuite/Magento/PaypalGraphQl/Model/Resolver/Customer/PlaceOrderWithPayflowProTest.php

```diff
@@ -77,6 +77,7 @@ public function testResolveCustomer(): void
         payment_method: {
           code: "{$paymentMethod}",
             payflowpro: {
+              is_active_payment_token_enabler: true
```

Comment:
Along with this please perform assertions validating if the payment data is being stored on the customer. You can replicate the test for true and false scenarios.